frontend2/src/api/loaders/homeIfLoggedIn.ts

```diff
@@ -3,7 +3,7 @@ import { loginCheck } from "api/auth/authApi";
 import { type LoaderFunction } from "react-router-dom";
 import { DEFAULT_EPISODE } from "utils/constants";
 
-export const passwordForgotLoader =
+export const homeIfLoggedIn =
```

Comment:
this is because I changed this to be used for login as well... know of any other pages that could use it?

Reply:
registration page? not sure if there's anything else